### DISCUSSION TITLE
AOC24 - Advent of Code 2024 implemented in Arturo

### DISCUSSION THREAD

**Discussion by RickBarretto** [arturo-lang/arturo#1815] [Show and tell]
The idea of this Thread is to discuss about the Advent of Code 2024 implemented in Arturo.
Based on our latest version of [AOC23](https://github.com/arturo-lang/arturo/discussions/1297).

Event
Challenges: [AOC24](https://adventofcode.com/)
Date: 1st, December » 25th, December
Hour: 12am, UTC-5

---

### Discord

We've created the same thread on Discord: [Thread: AOC 2024 - Advent of Code 2024 in Arturo](https://discord.com/channels/765519132186640445/1312757188569075733)

### Recommendations

The repository should use the tag `#aoc2024` and `#arturo`.
You can also show your repository in the comments' section bellow.

---

### Showcase
See other implementations on the [Index of Repositories](https://github.com/search?q=topic%3Aarturo+topic%3Aaoc2024&type=repositories).

Show us your own implementations on this Thread and have fun! 🙂🎉

**Comment by asumbek** [comment]
AOC for this year
https://github.com/mbekkomo/aoc-2024
